_config.yml

```diff
@@ -73,6 +73,9 @@ search:
   # Supports true or false (default)
   button: false
 
+# For copy button on code
+copy_button_on_code: true
```

Comment:
Little nitpicking here: I'd prefer `enable_copy_code_button` - `enable` indicates a boolean (true/default-false), `copy_code` is an action. More speaking to me.

Reply:
you're right